comprobo_final_project/scripts/models/robot.py

```diff
@@ -23,8 +23,10 @@ def __init__(self):
         self.resolution = 100
 
         # Suscribe to position of Neato robot
-        GazeboPoseProvider(rospy).subscribe(self._pose_listener) # For Gazebo
-        # AprilPoseProvider(rospy).subscribe(self._pose_listener) # For real world
+        if real is False:
```

Comment:
```
pose_provider = AprilPoseProvider(rospy) if real else GazeboPoseProvider(rospy)
```

I don't actually know if this would work, but it's cleaner